test/utils/format-date-test.js

```diff
@@ -90,9 +90,22 @@ describe('utils/format-date', () => {
   test({
     description: `current time after end time return 0`,
     assertions: () => {
-      const actual = getDaysRemaining(1520300344)
+      const actual = getDaysRemaining(1520300344, 1520300300)
 
       assert.strictEqual(actual, 0, `didn't return 0 as expected`)
     },
   })
+
+  test({
+    description: `given timestamp does format correct`,
```

Comment:
Are we concerned with `formattedLocalShort` and `formattedLocal` here?

Reply:
nope, get days remaining just returns whole days remaining.